package.json

```diff
@@ -131,7 +128,7 @@
     "rxjs": "~6.4.0",
     "sauce-connect": "https://saucelabs.com/downloads/sc-4.5.4-linux.tar.gz",
     "semver": "6.3.0",
-    "source-map": "^0.5.6",
+    "source-map": "^0.7.3",
```

Comment:
With this change we can close https://github.com/angular/angular-cli/issues/13228 and remove `source-map from the ignore list https://github.com/angular/angular-cli/blob/6f0e7bf437ae98d8d077c1f056f210fc407586f0/renovate.json#L21`

Reply:
Yes.  that would be a good followup to this.

Reply:
Actually, small change so just added it to that commit.